class.jetpack.php

```diff
@@ -4032,6 +4032,20 @@ function admin_page_load() {
 					wp_safe_redirect( Jetpack::admin_url( array( 'page' => $redirect ) ) );
 				}
 				exit;
+			case 'onboard' :
+				if ( ! current_user_can( 'manage_options' ) ) {
+					wp_safe_redirect( Jetpack::admin_url( 'page=jetpack' ) );
```

Comment:
If a user doesn't have the `manage_options` capability -- can they even access `/wp-admin/admin.php?page=jetpack` at all?

Reply:
Yes - even subscribers can access it. With limited features, of course, but they can still access it.

Reply:
Okay, cool 👍